styles/scss/main.scss

```diff
@@ -0,0 +1,148 @@
+@import "color";
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+html {
+    box-sizing: border-box;
+    font-size: 16px;
+}
+
+.container {
+    width: 1020px;
+    margin: 0 auto;
+}
+
+.button {
+    width: 45px;
+    height: 45px;
+    background-color: $mainColor;
+    border-radius: 5px;
+    text-align: center;
+    line-height: 10px;
+    border: 1px solid white;
+    margin-right: 5px;
+    font-family: 'Roboto', sans-serif;
+    color: white;
+    transition: 0.1s;
+    &:hover {
+        background-color: $hoverColor;
+        border: 1px solid white; 
+        color: white;
+        cursor: pointer;
+    }
+}
+
+.button-big{
+    width: 97px;
+    height: 45px;
+    background-color: $mainColor;
+    border-radius: 5px;
+    text-align: center;
+    line-height: 10px;
+    border: 1px solid white;
+    margin-right: 5px;
+    font-family: 'Roboto', sans-serif;
+    color: white;
+    &:hover {
+        background-color: $hoverColor;
+        border: 1px solid white; 
+        color: white;
+        cursor: pointer;
+    }
+}
+
+.button-big-space{
+    width: 357px;
+    height: 45px;
+    background-color: $mainColor;
+    border-radius: 5px;
+    text-align: center;
+    line-height: 10px;
+    border: 1px solid white;
+    margin-right: 5px;
+    font-family: 'Roboto', sans-serif;
+    color: white;
+    &:hover {
+        background-color: $hoverColor;
+        border: 1px solid white; 
+        color: white;
+        cursor: pointer;
+        translate: 1s liner;
+    }
+}
```

Comment:
into
```
.button-big{
          &-space {...}
}
```

Reply:
> into
> 
> ```
> .button-big{
>           &-space {...}
> }
> ```

Thank you for your comment. I tried to fix